application/src/main/java/org/togetherjava/tjbot/commands/example/CommandExample.java

```diff
@@ -47,17 +49,20 @@ public CommandExample() {
                 // * adds the second option
                 new OptionData(OptionType.STRING, "times-to-ping",
                         "Amount of times the user will be pinged, default 1")
-                            .addChoices(new Command.Choice("Once", "1"),
-                                    new Command.Choice("Twice", "2"))
+                            .addChoices(
```

Comment:
Not sure if this is formatter related but the deleted lines look more readable to me.

Reply:
I'd assume formatter since I don't remember it myself